jwst/assign_wcs/util.py

```diff
@@ -122,47 +192,63 @@ def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=Non
     """
     bb = bounding_box
     wcslist = [im.meta.wcs for im in dmodels]
+
     if not isiterable(wcslist):
         raise ValueError("Expected 'wcslist' to be an iterable of WCS objects.")
+
     if not all([isinstance(w, WCS) for w in wcslist]):
         raise TypeError("All items in wcslist are to be instances of gwcs.WCS.")
+
     if refmodel is None:
         refmodel = dmodels[0]
     else:
         if not isinstance(refmodel, DataModel):
             raise TypeError("Expected refmodel to be an instance of DataModel.")
 
-    fiducial = compute_fiducial(wcslist, bb)
+    fiducial = compute_fiducial(wcslist, bb)  # possibly retrieve from reference wcs
```

Comment:
What is this comment here?  Not sure the meaning or context.

Reply:
It was a note that I had from our conversation that I forgot to remove. Sorry about that